modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -130,10 +139,33 @@ const getOptions = (port: number, browserName: string, settings: DriverSettings,
     }
   }
 
+  // Remote webdriver settings
```

Comment:
This is kind of obvious. So the comment is not really needed.

Reply:
Removed this comment